remotetracker/github.go

```diff
@@ -0,0 +1,39 @@
+package remotetracker
+
+import (
+	"encoding/json"
+
+	"github.com/almighty/almighty-core/models"
+	"github.com/google/go-github/github"
+	"github.com/jinzhu/gorm"
+	"github.com/satori/go.uuid"
+)
+
+// fetch collects data from Github
+func fetchGithub(url, query string, item chan map[string]interface{}) {
+	client := github.NewClient(nil)
+	result, _, _ := client.Search.Issues(query, nil)
+	issues := result.Issues
+	for l := range issues {
+		i := make(map[string]interface{})
+		id, _ := json.Marshal(issues[l].URL)
+		description, _ := json.Marshal(issues[l].Body)
```

Comment:
This is just a subset of the field available in the GitHub issue. Can we not get to the raw map[string]interface{} from the GiHub response and use that directly instead?